cypress/support/api/data-import.js

```diff
@@ -0,0 +1,10 @@
+Cypress.Commands.add('getJobLogs', (searchParams) => {
+  cy
+    .okapiRequest({
+      path: 'metadata-provider/jobExecutions',
+      searchParams,
+    })
+    .then(({ body }) => {
+      Cypress.env('jobLogs', body.jobExecutions);
```

Comment:
think that support of dynamic test data through env variables is not a good idea. Do you have alternatives?

Reply:
Hi @dmitriyfolio . The reason we have to use dynamic data is because I need to search by `id`. The `id` is generated dynamically on the server so we do not know it beforehand.

Reply:
can you request related information from server into you test and keep it in scope of your test?

Reply:
You mean , should I move this block of code into my test file, right? I see the pattern in other tests. They all implement fetching data from server like that. So, I followed it

Reply:
could you give a link to such tests?

Reply:
I see a few examples of such work through sys variables. I think all these places are not good and we will repair it in the nearest future to something what I wrote into this chat. Reason - extra difficulty to support such tests and potential risk related with management of such variables.

Reply:
could you improve your code into this PR based on my suggestion?

Reply:
@dmitriyfolio , It is done